docs/site/content/docs/getting-started-tanzu-workloads.md

```diff
@@ -0,0 +1,236 @@
+```
+
+1. Install the Application Toolkit package.
+
+Application Toolkit package will install software required to create a running application from your source code.
+
+Application Toolkit will also prepare the `default` namespace so that you can start creating applications immediately. For preparing another namespace, please refer to the [Prepare your Developer Namespace](package-readme-app-toolkit-0.2.0.md#set-up-the-developer-namespace) section.
+
+```shell
+tanzu package install app-toolkit --package-name app-toolkit.community.tanzu.vmware.com --version 0.2.0 -f app-toolkit-values.yaml -n tanzu-package-repo-global
+```
+
+1. Verify the package is now installed.
+
+  ```sh
+  tanzu package installed list
+  ```
+
+  ```txt
+  NAME          PACKAGE-NAME                             PACKAGE-VERSION  STATUS
+  app-toolkit               app-toolkit.community.tanzu.vmware.com               0.2.0            Reconcile succeeded  tanzu-package-repo-global
+  cartographer              cartographer.community.tanzu.vmware.com              0.3.0            Reconcile succeeded  tanzu-package-repo-global
+  cartographer-catalog      cartographer-catalog.community.tanzu.vmware.com      0.3.0            Reconcile succeeded  tanzu-package-repo-global
+  cert-manager              cert-manager.community.tanzu.vmware.com              1.6.1            Reconcile succeeded  tanzu-package-repo-global
+  contour                   contour.community.tanzu.vmware.com                   1.20.1           Reconcile succeeded  tanzu-package-repo-global
+  fluxcd-source-controller  fluxcd-source-controller.community.tanzu.vmware.com  0.21.2           Reconcile succeeded  tanzu-package-repo-global
+  knative-serving           knative-serving.community.tanzu.vmware.com           1.0.0            Reconcile succeeded  tanzu-package-repo-global
+  kpack                     kpack.community.tanzu.vmware.com                     0.5.2            Reconcile succeeded  tanzu-package-repo-global
+  kpack-dependencies        kpack-dependencies.community.tanzu.vmware.com        0.0.9            Reconcile succeeded  tanzu-package-repo-global
+  cni                       calico.community.tanzu.vmware.com                    3.22.1           Reconcile succeeded  tkg-system
+  secretgen-controller      secretgen-controller.community.tanzu.vmware.com      0.7.1            Reconcile succeeded  tkg-system
+  ```
+
+## Create a Tanzu Workload
+
+1. We will use a sample git repo to demonstrate how to create a Tanzu workload from your source code using `tanzu apps`.
+  
+For using your own repository, please refer to Create a Tanzu workload section in Application Toolkit Package Docs.
+
+  ```shell
+  tanzu apps workload create hello-world \
+              --git-repo  https://github.com/vmware-tanzu/application-toolkit-sample-app \
+              --git-branch main \
+              --type web \
+              --label app.kubernetes.io/part-of=hello-world \
+              --yes \
+              --tail
```

Comment:
You could be explicit here by adding `--namespace default` to avoid any accidental user error from them having a different namespace targeted

Reply:
makes sense. incorporated!